sycl/test/e2e_test_requirements/no_sycl_hpp_in_e2e_tests.cpp

```diff
@@ -6,7 +6,7 @@
 // CHECK-DAG: README.md
 // CHECK-DAG: lit.cfg.py
 //
-// CHECK-NUM-MATCHES: 5
+// CHECK-NUM-MATCHES: 6
```

Comment:
Took me a moment to realize why this was increasing. I suppose this is a good reason to let it go up.